src/messaging/Flags.h

```diff
@@ -60,6 +61,7 @@ enum class MessageFlagValues : uint32_t
 
 enum class SendMessageFlags : uint16_t
 {
+    kSendFlag_Default = 0x0000,
```

Comment:
How about using `_None` instead of `_Default`? default seems to make it sound that there are some 'default known good settings' where as none is hinting more at 'nothing set'

Reply:
Acked